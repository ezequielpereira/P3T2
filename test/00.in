max_cantidad 
1
1-15
max_cantidad 
2
1-15
15-30
max_cantidad 
2
1-15
3-10
max_cantidad 
3
1-15
3-20
15-30
Fin
