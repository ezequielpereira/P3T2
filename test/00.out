1>Cantidad: 1
2>Cantidad: 2
3>Cantidad: 1
4>Cantidad: 2
5>Fin.
